src/Microsoft.ML.FastTree/Representation/TreeRegressorCollection.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML.Trainers.FastTree.Internal;
+
+namespace Microsoft.ML.FastTree.Representation
+{
+    /// <summary>
+    /// A list of <see cref="TreeRegressor"/>. To compute the output value of a <see cref="TreeRegressorCollection"/>, we need to compute
+    /// the output values of all trees in <see cref="Trees"/>, scale those values via <see cref="TreeWeights"/>, and finally sum the scaled
+    /// values and <see cref="Bias"/> up.
```

Comment:
I don't see such behavior in our code.
[https://github.com/dotnet/machinelearning/blob/b4c10662706589426a0a870304e3088dec8077ac/src/Microsoft.ML.FastTree/TreeEnsemble/TreeEnsemble.cs#L257](https://github.com/dotnet/machinelearning/blob/b4c10662706589426a0a870304e3088dec8077ac/src/Microsoft.ML.FastTree/TreeEnsemble/TreeEnsemble.cs#L257)
From what I see, we just sum up tree output.
And I don't see any usage of Bias or TreeWeight in code other than setter, or output as PFA or INI.

Not sure is it right or not.
@tfinley@gmail.com  Probably should knew more.
But if we don't use Weights and Bias, I concern we need need to expose them.